src/js/_enqueues/vendor/plupload/handlers.js

```diff
@@ -276,9 +276,12 @@ function switchUploader( s ) {
 
 		if ( typeof( uploader ) == 'object' )
 			uploader.refresh();
+		
+		jQuery( '#plupload-browse-button' ).trigger( 'focus' );
```

Comment:
```suggestion

		jQuery( '#plupload-browse-button' ).trigger( 'focus' );
```

minor edit: the empty line had extra tabs